models/edr/run_results/model_run_results.sql

```diff
@@ -36,6 +36,10 @@ SELECT
     models.path,
     models.original_path,
     models.owner,
-    models.alias
+    models.alias,
+    ROW_NUMBER() OVER (PARTITION BY run_results.unique_id ORDER BY run_results.generated_at DESC, run_results.execute_started_at DESC, run_results.execute_completed_at DESC ) AS model_invocation_index,
+    CASE WHEN DENSE_RANK() OVER (ORDER BY run_results.generated_at DESC) = 1 THEN TRUE ELSE FALSE END AS is_latest_invocation, 
+    CASE WHEN DENSE_RANK() OVER (PARTITION BY run_results.generated_at::date ORDER BY run_results.generated_at ASC) = 1 THEN TRUE ELSE FALSE END AS is_the_first_invocation_of_the_day
+    
```

Comment:
Notice that another model named `alerts_dbt_models` is based on `model_run_results` 👀 
Currently it does a union between `model_run_results` and `snapshot_run_results`, which is failing due to mismatched column numbers between the 2 models.

You can either:
1. Select the specific columns in the union at the model `alerts_dbt_models` (My preferred solution)
2.  Add those columns to the `snapshot_run_results` model

Reply:
Done :)